apps/heft/src/utilities/GitUtilities.ts

```diff
@@ -230,6 +293,131 @@ export class GitUtilities {
     }
   }
 
+  private async _getIgnoreMatchersAsync(gitRepoRootPath: string): Promise<Map<string, IIgnoreMatcher>> {
```

Comment:
Unit tests?

Reply:
Oops, forgot to include those. They were staged but not comitted.